app/src/main/java/com/simplecity/amp_library/ui/screens/songs/list/SongListFragment.kt

```diff
@@ -288,7 +288,7 @@ class SongListFragment :
 
     override fun onSongClick(position: Int, songView: SongView) {
         if (!contextualToolbarHelper!!.handleClick(songView, songView.song)) {
-            val uiEvent = EventUtils.newUiEvent(songView.song, UiEventType.PLAY, context)
+            val uiEvent = EventUtils.newUiEvent(songView.song, UiEventType.PLAY, context!!)
```

Comment:
For this and another invocations of the same method - if `context` can't be null, it's better to do a null check before executing this to avoid potentiall issues with null contexts. So:

```kotlin
if (context != null) {
    val uiEvent = EventUtils.newUiEvent(songView.song, UiEventType.PLAY, context)
    ...
````

`!!` will throw an NPE if `context` is null, so if we want to avoid NPEs we really shouldn't be using `!!`. See https://kotlinlang.org/docs/reference/null-safety.html.